Graphics/Blank.hs

```diff
@@ -343,19 +346,22 @@ blankCanvas opts actions = do
                ) app
 
 generalSend :: forall m a . RunMonad m
-            => (DeviceContext -> m Cmd Proc :~> IO) -> DeviceContext -> Canvas a -> IO a
+            => (DeviceContext -> m Prim :~> IO) -> DeviceContext -> Canvas a -> IO a
 generalSend f cxt (Canvas c) = do
     -- XXX: Is it ok to hardcode 0 as the start value here?
     -- AJG: No, its not.
-   let m0 :: RemoteMonad Cmd Proc a
+   let m0 :: RemoteMonad Prim a
        m0 = evalStateT (runReaderT c (deviceCanvasContext cxt)) 0
    runMonad (f cxt) N.# m0
-
-sendS, sendW :: DeviceContext -> Canvas a -> IO a
+{-
+sendS :: DeviceContext -> Canvas a -> IO a
```

Comment:
There's lots of commented-out code in this module, such as `sendS`, `sendS'`, and some imports/exports. Can these be removed?